charts/spiderpool/files/grafana-rdma-cluster.json

```diff
@@ -97,7 +97,7 @@
                         "uid": "${datasource}"
                     },
                     "editorMode": "code",
-                    "expr": "count(count(rdma_tx_vport_rdma_unicast_bytes_total{pod_name!=\"\"}) by (node_name))",
+                    "expr": "count(count(rdma_tx_vport_rdma_unicast_bytes_total{pod_name!=\"\",cluster=~\"$cluster\"}) by (node_name))",
```

Comment:
I suggest it could add a new helm value , which could help turn on or off the cluster label . the off case is used for open-source case